fleet/fleet.go

```diff
@@ -131,6 +132,15 @@ type Fleet interface {
 func NewFleet(config Config) (Fleet, error) {
 	var trans http.RoundTripper
 
+	tunneling := config.Tunnel.Tunnel != ""
+	// If a tunnel in provided we need to overwrite the http.Transport.Dial function
+	// to use the tunnel
+	if tunneling {
+		trans = &http.Transport{
+			Dial: config.Tunnel.NewDialFunc(config),
+		}
+	}
```

Comment:
https://github.com/giantswarm/inago/blob/ssh-tunnel/fleet/fleet.go#L167
the if statement in the `unix` case makes sure that the dail function is not overwritten in case tunneling is enabled.